src/pybind/mgr/dashboard/services/access_control.py

```diff
@@ -432,6 +434,22 @@ def get_role(self, name):
                 raise RoleDoesNotExist(name)
             return self.roles[name]
 
+    def increment_attempt(self, username, success=False):
```

Comment:
What do you think about renaming the method `increment_attempt` to `set_attempt` or similar? - Because if `success=True` it doesn't increment `attempt`. I hope you don't mind my nitpick.

Reply:
That makes sense, thank you. Updated the PR

Reply:
Thank you! :smile: